### PR TITLE
[django3] 📦 Update Python packages to support Django 3

### PR DESCRIPTION
**Pull request by wlonk**
Current blockers:

- ~django-rq==2.1.0 (https://github.com/rq/django-rq/pull/384)~
  - ~Depends on https://github.com/niwinz/django-redis/pull/394~
    - ~Depends on https://github.com/niwinz/django-redis/pull/387~
- ~django-parler (https://github.com/django-parler/django-parler/pull/255)~

cat

### CODE REVIEW

**Comment by wlonk** [comment]
[Support Django 3](https://trello.com/c/qLIwEmYE/122-support-django-3)

File: requirements/prod.in [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Django==2.2.9
+Django==3.0.0
```
Comment (davisagli) [inline requirements/prod.in]: latest is 3.0.2

File: requirements/prod.in [hunk 2]
```diff
@@ -25,6 +25,6 @@ psycopg2-binary==2.8.4
 pyOpenSSL==19.1.0
 sentry-sdk==0.13.5
 service_identity==18.1.0
-https://github.com/SFDO-Tooling/sfdo-template-helpers/archive/v0.9.0.tar.gz
+https://github.com/SFDO-Tooling/sfdo-template-helpers/archive/master.tar.gz
```
Comment (davisagli) [inline requirements/prod.in]: Seems safest to keep this pinned so that builds are consistent.

Comment (wlonk) [reply requirements/prod.in]: Can you tag v0.10.0 then? There are changes in master that make this work: https://github.com/SFDO-Tooling/sfdo-template-helpers/compare/v0.9.0...master

Comment (davisagli) [reply requirements/prod.in]: Oh I see. I'll do that soon; this is fine for now.

**Comment by jgerigmeyer** [comment]
@wlonk Since this change, when I `^-C` to stop the running Django server, I get:

```
^Cid=no-job-id at=INFO time="2020-01-07 16:21:16.337150" tag=external module=worker msg="Warm shut down requested"

# Traceback (most recent call last):
  File "manage.py", line 25, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.8/site-packages/newrelic/hooks/framework_django.py", line 973, in _nr_wrapper_BaseCommand_run_from_argv_
    return wrapped(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 341, in run_from_argv
    connections.close_all()
  File "/usr/local/lib/python3.8/site-packages/django/db/utils.py", line 230, in close_all
    connection.close()
  File "/usr/local/lib/python3.8/site-packages/django/utils/asyncio.py", line 24, in inner
    raise SynchronousOnlyOperation(message)
django.core.exceptions.SynchronousOnlyOperation: You cannot call this from an async context - use a thread or sync_to_async.
```

Not a big problem, but wanted to note in case there's an easy fix.

**Comment by jgerigmeyer** [comment]
@wlonk Oops; turns it this might be somewhat blocking after all. When I have the Django server running and save any change to a Py file (triggering a soft reload), the server crashes:

```
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 328, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/commands/runserver.py", line 60, in execute
    super().execute(*args, **options)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 369, in execute
    output = self.handle(*args, **options)
  File "/usr/local/lib/python3.8/site-packages/newrelic/api/function_trace.py", line 137, in literal_wrapper
    return wrapped(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/channels/management/commands/runserver.py", line 59, in handle
    super().handle(*args, **options)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/commands/runserver.py", line 95, in handle
    self.run(**options)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/commands/runserver.py", line 102, in run
    autoreload.run_with_reloader(self.inner_run, **options)
  File "/usr/local/lib/python3.8/site-packages/django/utils/autoreload.py", line 599, in run_with_reloader
    start_django(reloader, main_func, *args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/utils/autoreload.py", line 584, in start_django
    reloader.run(django_main_thread)
  File "/usr/local/lib/python3.8/site-packages/django/utils/autoreload.py", line 299, in run
    self.run_loop()
  File "/usr/local/lib/python3.8/site-packages/django/utils/autoreload.py", line 305, in run_loop
    next(ticker)
  File "/usr/local/lib/python3.8/site-packages/django/utils/autoreload.py", line 353, in tick
    self.notify_file_changed(filepath)
  File "/usr/local/lib/python3.8/site-packages/django/utils/autoreload.py", line 328, in notify_file_changed
    trigger_reload(path)
  File "/usr/local/lib/python3.8/site-packages/django/utils/autoreload.py", line 223, in trigger_reload
    sys.exit(3)
SystemExit: 3

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "manage.py", line 25, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 401, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.8/site-packages/django/core/management/__init__.py", line 395, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.8/site-packages/newrelic/hooks/framework_django.py", line 973, in _nr_wrapper_BaseCommand_run_from_argv_
    return wrapped(*args, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/django/core/management/base.py", line 341, in run_from_argv
    connections.close_all()
  File "/usr/local/lib/python3.8/site-packages/django/db/utils.py", line 230, in close_all
    connection.close()
  File "/usr/local/lib/python3.8/site-packages/django/utils/asyncio.py", line 24, in inner
    raise SynchronousOnlyOperation(message)
django.core.exceptions.SynchronousOnlyOperation: You cannot call this from an async context - use a thread or sync_to_async.
```

**Pull request merged**